api.bs

```diff
@@ -978,13 +1022,13 @@ and <a dictionary lt=PrivateAttributionConversionOptions>|options|</a>:
 1. Validate the page-supplied API inputs:
     1.  Switch on the value of <a dict-member for=PrivateAttributionConversionOptions>logic</a>:
         <dl class="switch">
-            :   <a enum-value for=PrivateAttributionLogic>"last-touch"</a>
-            ::  Perform the following steps:
-                1.  If <a dict-member for=PrivateAttributionConversionOptions>value</a> is 0,
-                    throw a {{RangeError}}.
-                1.  If <a dict-member for=PrivateAttributionConversionOptions>value</a>
-                    is greater than <a dict-member for=PrivateAttributionConversionOptions>maxValue</a>,
-                    throw a {{RangeError}}.
+        :   <a enum-value for=PrivateAttributionLogic>"last-touch"</a>
+        ::  Perform the following steps:
+            1.  If <a dict-member for=PrivateAttributionConversionOptions>value</a> is 0,
+                throw a {{RangeError}}.
+            1.  If <a dict-member for=PrivateAttributionConversionOptions>value</a>
+                is greater than <a dict-member for=PrivateAttributionConversionOptions>maxValue</a>,
+                throw a {{RangeError}}.
```

Comment:
Looks like something about the indentation on the following line is causing the build to fail.

Reply:
I hadn't rebased all the changes yet.  I've done so now.